app/src/main/java/org/oppia/android/app/devoptions/platformparameters/PlatformParametersFragmentPresenter.kt

```diff
@@ -174,6 +178,43 @@ class PlatformParametersFragmentPresenter @Inject constructor(
     editText.setTag(R.id.platform_parameter_text_watcher, newWatcher)
   }
 
+  private fun handleResetParam(
```

Comment:
Aligning with feature flag naming.
```suggestion
  private fun handleResetPlatformParameter(
```

Reply:
updated

Reply:
Can you double-check, I still see it as `handleResetParam`?

Reply:
updated now

Reply:
@theayushyadav11 -- can you please check again if it's fully updated? 
>handleReset**Platform**Parameter
https://github.com/oppia/oppia-android/blob/218bf32f7547c03876db2c013501811dd2f80d4e/app/src/main/java/org/oppia/android/app/devoptions/platformparameters/PlatformParametersFragmentPresenter.kt#L253
If I am missing something, please add link to the changes here to verify.

Reply:
Updated now.
https://github.com/oppia/oppia-android/blob/5cb55387f536044a138653d8077add3d45f037bd/app/src/main/java/org/oppia/android/app/devoptions/platformparameters/PlatformParametersFragmentPresenter.kt#L253